magnolia/shared/src/main/scala-dotty/zio/config/magnolia/Descriptor.scala

```diff
@@ -0,0 +1,165 @@
+package zio.config.magnolia
```

Comment:
Even if it gives a bit of performance drill down, move away from using Strings in arguments. Example: Use List[SubClassName] instead of List[string]

Reply:
Done